web/src/pages/User/Login.less

```diff
@@ -30,14 +30,13 @@
   text-align: right;
 }
 
-.settings {
-  display: inline-flex;
-  align-items: center;
-  justify-content: center;
-  padding: 12px;
-  font-size: 16px;
-  vertical-align: middle;
-  cursor: pointer;
+a{
+  color: black;
+  
+  &:hover {
+    color: @primary-color;
+  }
+
 }
```

Comment:
Hi @iamayushdas 

Thanks for your contribution, I think the `a` to should add the space.